src/Bridge/Symfony/Bundle/Resources/public/swagger-ui/oauth2-redirect.html

```diff
@@ -0,0 +1,67 @@
+<!doctype html>
```

Comment:
The `update-js.sh` script must be upgraded to copy the new version of this file every time we'll bump Swagger UI: https://github.com/api-platform/core/blob/master/update-js.sh#L12